### PR TITLE
Update UWP toolchain version

### PR DESCRIPTION
**Pull request by Manishearth**
I've been working with these patches applied for a while, they seem to work great.

r? @jdm

### CODE REVIEW

File: support/hololens/ServoApp/DefaultUrl.h [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #pragma once
 
-#define DEFAULT_URL L"https://servo.org/hl-home/"
+#define DEFAULT_URL L"https://manishearth.net/sand/vr.html"
```
Comment (jdm) [inline support/hololens/ServoApp/DefaultUrl.h]: Revert this?

**Review by jdm — CHANGES_REQUESTED** [review]

File: support/hololens/ServoApp/DefaultUrl.h (hunk 1 above)
Comment (Manishearth) [reply support/hololens/ServoApp/DefaultUrl.h]: oops, done

**Comment by CYBAI** [comment]
```
Build started 3/12/2020 8:43:13 AM.
     1>Project "c:\users\task_1~1\appdata\local\temp\tmpewsqbc" on node 1 (default targets).
     1>Project "c:\users\task_1~1\appdata\local\temp\tmpewsqbc" (1) is building "C:\Users\task_1584001125\repo\support\hololens\ServoApp.sln" (2) on node 1 (Build target(s)).
     2>ValidateSolutionConfiguration:
         Building solution configuration "Debug|arm64".
     2>Project "C:\Users\task_1584001125\repo\support\hololens\ServoApp.sln" (2) is building "C:\Users\task_1584001125\repo\support\hololens\ServoApp\ServoApp.vcxproj" (3) on node 1 (default targets).
     3>C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\VC\VCTargets\Microsoft.Cpp.WindowsSDK.targets(46,5): error MSB8036: The Windows SDK version 10.0 was not found. Install the required version of Windows SDK or change the SDK version in the project property pages or by right-clicking the solution and selecting "Retarget solution". [C:\Users\task_1584001125\repo\support\hololens\ServoApp\ServoApp.vcxproj]
     3>Done Building Project "C:\Users\task_1584001125\repo\support\hololens\ServoApp\ServoApp.vcxproj" (default targets) -- FAILED.
     2>Done Building Project "C:\Users\task_1584001125\repo\support\hololens\ServoApp.sln" (Build target(s)) -- FAILED.
     1>Done Building Project "c:\users\task_1~1\appdata\local\temp\tmpewsqbc" (default targets) -- FAILED.
Build FAILED.
       "c:\users\task_1~1\appdata\local\temp\tmpewsqbc" (default target) (1) ->
       "C:\Users\task_1584001125\repo\support\hololens\ServoApp.sln" (Build target) (2) ->
       "C:\Users\task_1584001125\repo\support\hololens\ServoApp\ServoApp.vcxproj" (default target) (3) ->
       (_CheckWindowsSDKInstalled target) -> 
         C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\VC\VCTargets\Microsoft.Cpp.WindowsSDK.targets(46,5): error MSB8036: The Windows SDK version 10.0 was not found. Install the required version of Windows SDK or change the SDK version in the project property pages or by right-clicking the solution and selecting "Retarget solution". [C:\Users\task_1584001125\repo\support\hololens\ServoApp\ServoApp.vcxproj]
    0 Warning(s)
    1 Error(s)
Time Elapsed 00:00:00.36
```

File: support/hololens/ServoApp/ServoApp.vcxproj [hunk 2]
```diff
@@ -13,8 +13,8 @@
     <AppContainerApplication>true</AppContainerApplication>
     <ApplicationType>Windows Store</ApplicationType>
     <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
-    <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.17763.0</WindowsTargetPlatformVersion>
-    <WindowsTargetPlatformMinVersion>10.0.17763.0</WindowsTargetPlatformMinVersion>
+    <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0</WindowsTargetPlatformVersion>
```
[the remark this answers is not in the document]
Comment (Manishearth) [reply support/hololens/ServoApp/ServoApp.vcxproj]: hmm, this was autogenerated by VS. But it should be 10.0.18something

**Comment by jdm** [comment]
I'm building a new AMI that installs the newest SDK.

**Comment by jdm** [comment]
Same issue with the 10.18whatever SDK explicitly installed in the windows AMI. I recommend modifying that 10.0 by hand to match the min version.

**Comment by Manishearth** [comment]
```
Build started 3/16/2020 5:23:50 AM.
     1>Project "c:\users\task_1~1\appdata\local\temp\tmpdysbwd" on node 1 (default targets).
     1>Project "c:\users\task_1~1\appdata\local\temp\tmpdysbwd" (1) is building "C:\Users\task_1584321971\repo\support\hololens\ServoApp.sln" (2) on node 1 (Build target(s)).
     2>ValidateSolutionConfiguration:
         Building solution configuration "Debug|x64".
     2>Project "C:\Users\task_1584321971\repo\support\hololens\ServoApp.sln" (2) is building "C:\Users\task_1584321971\repo\support\hololens\ServoApp\ServoApp.vcxproj" (3) on node 1 (default targets).
     3>C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\VC\VCTargets\Microsoft.Cpp.WindowsSDK.targets(46,5): error MSB8036: The Windows SDK version 10.0.18362.0 was not found. Install the required version of Windows SDK or change the SDK version in the project property pages or by right-clicking the solution and selecting "Retarget solution". [C:\Users\task_1584321971\repo\support\hololens\ServoApp\ServoApp.vcxproj]
     3>Done Building Project "C:\Users\task_1584321971\repo\support\hololens\ServoApp\ServoApp.vcxproj" (default targets) -- FAILED.
     2>Done Building Project "C:\Users\task_1584321971\repo\support\hololens\ServoApp.sln" (Build target(s)) -- FAILED.
     1>Done Building Project "c:\users\task_1~1\appdata\local\temp\tmpdysbwd" (default targets) -- FAILED.
```

@jdm  we still need this installed it seems

**Comment by Manishearth** [comment]
```
Build started 3/17/2020 3:27:09 AM.
     1>Project "c:\users\tab7f6~1\appdata\local\temp\tmpi0qn00" on node 1 (default targets).
     1>Project "c:\users\tab7f6~1\appdata\local\temp\tmpi0qn00" (1) is building "C:\Users\task_1584405424\repo\support\hololens\ServoApp.sln" (2) on node 1 (Build target(s)).
     2>ValidateSolutionConfiguration:
         Building solution configuration "Debug|arm64".
     2>Project "C:\Users\task_1584405424\repo\support\hololens\ServoApp.sln" (2) is building "C:\Users\task_1584405424\repo\support\hololens\ServoApp\ServoApp.vcxproj" (3) on node 1 (default targets).
     3>C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\VC\VCTargets\Microsoft.Cpp.WindowsSDK.targets(46,5): error MSB8036: The Windows SDK version 10.0.18362.0 was not found. Install the required version of Windows SDK or change the SDK version in the project property pages or by right-clicking the solution and selecting "Retarget solution". [C:\Users\task_1584405424\repo\support\hololens\ServoApp\ServoApp.vcxproj]
     3>Done Building Project "C:\Users\task_1584405424\repo\support\hololens\ServoApp\ServoApp.vcxproj" (default targets) -- FAILED.
     2>Done Building Project "C:\Users\task_1584405424\repo\support\hololens\ServoApp.sln" (Build target(s)) -- FAILED.
     1>Done Building Project "c:\users\tab7f6~1\appdata\local\temp\tmpi0qn00" (default targets) -- FAILED.
```

same error

**Comment by Manishearth** [comment]
same error again

**Comment by jdm** [comment]
I'm going to try upgrading CI to use VS 2019 and give this another shot.

**Comment by Manishearth** [comment]
Oh! Yeah, I think this toolchain may not exist in VS 2017

**Comment by Manishearth** [comment]
```
     3>C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(777,5): error : The OutputPath property is not set for project 'ServoApp.vcxproj'.  Please check to make sure that you have specified a valid combination of Configuration and Platform for this project.  Configuration='Debug'  Platform='ARM64'.  You may be seeing this message because you are trying to build a project without a solution file, and have specified a non-default Configuration or Platform that doesn't exist for this project. [C:\Users\task_1587405298\repo\support\hololens\ServoApp\ServoApp.vcxproj]
```

huh

**Comment by jdm** [comment]
```
         Building solution configuration "Debug|arm64".
     2>Project "C:\Users\task_1587405298\repo\support\hololens\ServoApp.sln" (2) is building "C:\Users\task_1587405298\repo\support\hololens\ServoApp\ServoApp.vcxproj" (3) on node 1 (default targets).
     3>C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(777,5): error : The OutputPath property is not set for project 'ServoApp.vcxproj'.  Please check to make sure that you have specified a valid combination of Configuration and Platform for this project.  Configuration='Debug'  Platform='ARM64'.  You may be seeing this message because you are trying to build a project without a solution file, and have specified a non-default Configuration or Platform that doesn't exist for this project. [C:\Users\task_1587405298\repo\support\hololens\ServoApp\ServoApp.vcxproj]
   
```
That's a new failure! Progress?

**Comment by jdm** [comment]
It's a failure that's present on master with the new AMI, so I'm trying to fix it in https://github.com/servo/servo/pull/26234.

**Comment by Manishearth** [comment]
@bors-servo retry

 - #25513

**Pull request merged**